tests/storages_tests/rdb_tests/test_storage.py

```diff
@@ -42,6 +42,7 @@ def test_init() -> None:
 
     assert storage.get_current_version() == storage.get_head_version()
     assert storage.get_all_versions() == [
+        "v3.0.0.d",
```

Comment:
I guess we need to update the following version lists:
- https://github.com/optuna/optuna/blob/ebbf70a608e3eab52f882befc3b65a7ba52d1711/tests/storages_tests/rdb_tests/test_storage.py#L176-L179
- https://github.com/optuna/optuna/blob/ebbf70a608e3eab52f882befc3b65a7ba52d1711/tests/storages_tests/rdb_tests/test_storage.py#L224
- https://github.com/optuna/optuna/blob/ebbf70a608e3eab52f882befc3b65a7ba52d1711/tests/storages_tests/rdb_tests/test_storage.py#L268

Reply:
Thanks! I added "3.0.0.d".

Reply:
Thank you!